src/lib/locale.store.ts

```diff
@@ -6,6 +6,8 @@ import fallback from '$locales/us/en/index.json';
 export const stringsSchema = z
   .object({
     alreadyHaveAnAccount: z.string(),
+    backToDefault: z.string(),
```

Comment:
Maybe a silly question but why do we need backToDefault and backToLogin? How are they different?

Reply:
Ok that makes sense now - thanks!